backend/tournesol/migrations/0053_entitypollrating_sum_trust_scores.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.9 on 2023-05-22 20:25
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tournesol', '0052_alter_entity_last_metadata_request_at'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='entitypollrating',
+            name='sum_trust_scores',
+            field=models.FloatField(default=0., help_text='Sum of trust scores of the contributors who rated the entity'),
```

Comment:
All recommendations will be reset as unsafe when this branch is deployed.  

Should we add a custom operation in this migration to compute the values (e.g with `EntityPollRating.bulk_update_sum_trust_scores(poll)`),  without waiting for the next ml run?

Reply:
As we quickly discussed. I propose to cut the PR in two, first adding the sum_trust_scores and their computations, then usingthe sum_trust_scores once they have been computed